konfera/settings.py

```diff
@@ -11,9 +11,31 @@
                             (45, _('45 min')),
                         ))
 
-"""LANDING PAGE is a composite of two keywords: <timewise>_<event>
+"""
+LANDING PAGE is a composite of two keywords: <timewise>_<event>
 <timewise> can be: latest or earliest
 <event> can be (at the moment): conference or meetup
 possible combinations: latest_conference (DEFAULT), latest_meetup, earliest_conference, earliest_meetup
 """
 LANDING_PAGE = getattr(settings, 'LANDING_PAGE', 'latest_conference')
+
+"""
+Register email notification.
+"""
+REGISTER_EMAIL_NOTIFY = getattr(settings, 'REGISTER_EMAIL_NOTIFY', False)
+REGISTER_EMAIL_BCC = getattr(settings, 'REGISTER_EMAIL_BCC', [])
+REGISTER_EMAIL = getattr(settings, 'REGISTER_EMAIL',
+                         'Dear {first_name} {last_name},\n\n' \
```

Comment:
I was thinking about it, and it is better idea to have it in file (or in DB), as I assume we will have more mail templates and some of them might have longer text. However the difficult thing is how to do it in a way that it can be easily overloaded. I imagine that someone do pip install django-konfera (once it is there :)), and the question here is how do we let admin modify the template? I dont like the idea that he would edit file in installed app, neither he will have fake app loaded before ours to load the overloaded templates first, we might look in project for specific dir if it exists and if the file is there, or the cleanest way that occured to me was to create a DB schema that will prepopulate the templates and user can edit them in admin area, but seemd like a lot of work (since we want launch tickets this week), therefore i went with setting as easy way since you can do local setting ad have your modifications there. My suggestion is let have a wider discussion about it and create a separate issue for this? If you have better suggestion, please let me know as this should be super easy for app admin to use.